notebooks/curation_api/python/get_collection_versions.ipynb

```diff
@@ -0,0 +1,130 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "c5490d93",
+   "metadata": {},
+   "source": [
+    "# Fetch all published versions of a Collection with full Dataset metadata"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "74560383",
+   "metadata": {},
+   "source": [
+    "The script in this notebook fetches full Collection metadata for all published versions of a Collection.\n",
+    "\n",
+    "Fetching versions of a Collection requires only the Collection id, and not an API key/access token."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0009d2d0",
+   "metadata": {},
+   "source": [
+    "### Import dependencies"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "aadd3c12",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from src.utils.config import set_api_access_config  # still required for setting api url env vars\n",
+    "from src.utils.logger import set_log_level  # can set_log_level(\"ERROR\") for less logging; default is \"INFO\"\n",
+    "from src.collection import get_collection_versions"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "928aa50e",
+   "metadata": {},
+   "source": [
+    "#### <font color='#bc00b0'>Please fill in the required values:</font>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4816c4cc",
+   "metadata": {},
+   "source": [
+    "<font color='#bc00b0'>(Required) Enter the id of the Collection for which you want to fetch all versions</font>\n",
+    "\n",
+    "_The Collection id can be found by looking at the url path in the address bar \n",
+    "when viewing your Collection in the CZ CELLxGENE Discover data portal: `/collections/{collection_id}`._"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bb82f0f9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "collection_id = \"01234567-89ab-cdef-0123-456789abcdef\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9501afeb",
+   "metadata": {},
+   "source": [
+    "### Set url env vars"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a6aab8e0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "set_api_access_config(env=\"dev\")"
```

Comment:
nit: for the other notebooks, we've been keeping prod as the default arg. Not against doing dev instead, but just pointing out the divergence in case we want to switch over for all of them

Reply:
thanks for catching this!!